docs/source/examples/lre-zne-comparison.md

```diff
@@ -0,0 +1,173 @@
+                                        trials=50,
+                                        seed=4)
+
+print(circuits[0])
+```
+
+We define an [executor](../guide/executors.md) which simulates the input circuit subjected to depolarizing noise, and returns the probability of measuring the ground state.
+By altering the value for `noise_level`, ideal and noisy expectation values can be obtained.
+
+```{code-cell} ipython3
+from cirq import DensityMatrixSimulator, depolarize
+
+def execute(circuit, noise_level=0.025):
+    noisy_circuit = circuit.with_noise(depolarize(p=noise_level))
+    rho = DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix
+    return rho[0, 0].real
+```
+
+```{code-cell} ipython3
+import numpy as np
+
+# Collect ideal and noisy values (probability of measuring 0 across all 100 circuits).
+noisy_values = np.zeros(len(circuits))
+ideal_values = np.zeros(len(circuits))
+
+for i, circuit in enumerate(circuits):
+    noisy = execute(circuit)
+    noisy_values[i] = noisy
+    ideal = execute(circuit, noise_level=0.0)
+    ideal_values[i] = ideal
+```
+
+```{code-cell} ipython3
+# The theoretical value for the probability of measuring 0 when taking an average over all the rotated rb circuits.
+p = lambda theta: 1 - (2/3) * np.sin(theta/2)**2
+
+print(f'Average error for noisy values: {np.abs(np.mean(noisy_values) - p(0.7))}')
+print(f'Average error for ideal values: {np.abs(np.mean(ideal_values) - p(0.7))}')
+```
+
+For the ideal values we still see a small error, since the circuits are randomly generated and the theoretical value is achieved in the limit where we generate all the possible circuits.
+
++++
+
+## Apply LRE and ZNE directly
+
+With the circuit and executor defined, we just need to choose the polynomial extrapolation degree as well as the fold multiplier. For ZNE we use the default values for the scale factors.
+
+```{code-cell} ipython3
+from mitiq.lre import execute_with_lre
+from mitiq.zne import execute_with_zne
+
+degree = 2
+fold_multiplier = 3
+
+# Collect mitigated values (probability of measuring 0 across all 100 circuits) using LRE and ZNE.
+mitigated_values_lre = np.zeros(len(circuits))
+mitigated_values_zne = np.zeros(len(circuits))
+
+for i, circuit in enumerate(circuits):
+    mitigated_lre = execute_with_lre(circuit,
+                                     execute,
+                                     degree=degree,
+                                     fold_multiplier=fold_multiplier)
+    mitigated_values_lre[i] = mitigated_lre
+    mitigated_zne = execute_with_zne(circuit, execute)
+    mitigated_values_zne[i] = mitigated_zne
+```
+
+```{code-cell} ipython3
+error_lre = np.abs(np.mean(mitigated_values_lre) - p(0.7))
+error_zne = np.abs(np.mean(mitigated_values_zne) - p(0.7))
+
+print(f'Average error of mitigated values using LRE: {error_lre}')
+print(f'Average error of mitigated values using ZNE: {error_zne}')
+```
+
+## Resource estimation
```

Comment:
We can discuss this in more detail, but I think it is nice to use these new functions to access the circuits that are to be executed.

Reply:
Met with Farrokh to discuss today and we agreed the tutorial as written is good. The resource estimation workflow is good to demo on its own, even if the circuits have to be regenerated. It demonstrates a more realistic workflow, rather than running, and then looking at overhead.